cmd/minikube/cmd/dashboard.go

```diff
@@ -53,19 +53,20 @@ var dashboardCmd = &cobra.Command{
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
 
-		url, err := cluster.GetServiceURL(api, namespace, service, nil)
+		urls, err := cluster.GetServiceURLs(api, namespace, service, nil)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			fmt.Fprintln(os.Stderr, "Check that minikube is running.")
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
-		if dashboardURLMode {
-			fmt.Fprintln(os.Stdout, url)
-		} else {
-			fmt.Fprintln(os.Stdout, "Opening kubernetes dashboard in default browser...")
-			browser.OpenURL(url)
+		for _, url := range urls {
```

Comment:
We should only have one URL for the dashboard right, could we just check the length, warn if there is more than one and continue like we used to without the loop?